src/Assetic/Extension/Twig/TwigFormulaLoader.php

```diff
@@ -34,7 +34,7 @@ public function __construct(\Twig_Environment $twig, LoggerInterface $logger = n
     public function load(ResourceInterface $resource)
     {
         try {
-            $tokens = $this->twig->tokenize($resource->getContent(), (string) $resource);
+            $tokens = $this->twig->tokenize(new \Twig_Source($resource->getContent(), (string) $resource), (string) $resource);
```

Comment:
I believe the second argument of `tokenize` is not needed anymore when you supply a `Twig_Source` argument.